### DISCUSSION TITLE
Web error "error loading dynamically imported module"

### DISCUSSION THREAD

**Discussion by paulmorabito** [immich-app/immich#7340] [Q&A] [answered]
### The bug

After clearing cache and all browser info, logging in to Immich is fine. Refreshing the page though produces this result:

error loading dynamically imported module: https://xx.xxx.net/_app/immutable/nodes/21.A5foPmpu.js (500)

After this, i'm unable to do anything except open Immich in an incognito window or clear browser cache etc. all over again.

This has only began occurring on recent versions but is present up to the current release (1.95.1).

### The OS that Immich Server is running on

Debian 12

### Version of Immich Server

1.95.1

### Version of Immich Mobile App

1.95.0 build 139

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
I can provide if needed
```

### Your .env content

```Shell
I can provide if needed
```

### Reproduction steps

```bash
1. Log in to Immich on Chrome, Firefox, Safari.
2. Reload page.
3. Error above appears.
```

**Comment by bo0tzz** [comment]
Can you try in an incognito window, another browser, etc?

**Comment by paulmorabito** [comment]
> Can you try in an incognito window, another browser, etc?

That always works first time and then after hitting refresh it starts again. I've tried multiple browsers, across OS's, and its always the same.

**Comment by bo0tzz** [comment]
Are you using a reverse proxy? If so, does it work without?

**Comment by paulmorabito** [comment]
> Are you using a reverse proxy? If so, does it work without?

Ah yes, I am using traefik. It doesn't happen when I access directly via IP:PORT address.

**Reply by bo0tzz** [reply]
Then it's probably Traefik wrongly caching something.

**Reply by paulmorabito** [reply, marked as answer]
Yes, solution is [here](https://github.com/immich-app/immich/issues/6057#issuecomment-1877608277).
